util/execdetails/execdetails.go

```diff
@@ -124,9 +133,20 @@ type RuntimeStats struct {
 	rows int64
 }
 
+// CopRuntimeStats collects info of one executor executed on TiKV.
```

Comment:
Why don't we reuse RuntimeStats?

Reply:
Good suggestion.
I have updated it and `RuntimeStats` is reused now.